### ISSUE TITLE
keyboard shortcuts are not saved

### ISSUE THREAD

**Issue by samhuss** [zen-browser/desktop#1748]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

Shortcuts are not saved after changing them in keyboard shortcuts settings window.

I've been looking at this file: ZenKeyboardshortcuts.mjs and notices that there are 2 variables for the default saved shortcuts:

_savedShortcuts
__savedShortcuts 

When the default shortcuts are loaded they are saved into the one with double underscore, while the rest of the functions read/write to the one with single _

I changed the mode of file: zen-keyboard-shortcuts.json under default profile to read only after editing the file with the required shortcuts and they got loaded properly in the settings page, however the new shortcut didn't take effect as seems browser memory is reading from the runtime default settings not the one saved from the previous session. 

I haven't dug deeper into the code but thought that might help mitigating the issue. 

https://github.com/zen-browser/components/blob/36e62781e27c19fade2df4d6a63e9062862ce7b0/src/ZenKeyboardShortcuts.mjs#L201

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.5 (64-bit)

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by R3flector** [comment]
same on arch

**Comment by realecto** [comment]
Can confirm the same on Windows 10

**Comment by thebiblelover7** [comment]
Seems like a duplicate of #1715 to me.

**Comment by mauro-balades** [comment]
This has been fixed for the next release!
